src/os/bluestore/BlueFS.h

```diff
@@ -723,5 +709,22 @@ class FitToFastVolumeSelector : public OriginalVolumeSelector {
 
   void get_paths(const std::string& base, paths& res) const override;
 };
-
+/**
+ * Directional graph of locks.
+ * Vertices - Locks. Edges (directed) - locking progression.
+ * Edge A->B exist if last taken lock was A and next taken lock is B.
+ * 
+ * Column represents last lock taken.
```

Comment:
Shouldn't Column and Row be exchanged in these two sentences?
Isn't it a ROW which represent last taken lock?
E.g. we have  L(row)->N(col) transition indicating the following sequence:
 std::lock_guard ll(log.lock);
 std::lock_guard nl(nodes.lock);